spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/PubSubAcknowledger.java

```diff
@@ -22,6 +22,7 @@
  * Acknowledges Pub/Sub messages based on their subscription name and ack ID.
  *
  * @author João André Martins
+ * @author Doug Hoard
```

Comment:
Does looks like you have done anything to this class 😃

Reply:
Forgot to revert that ... will fix :)